docs/jackson/deploy/env-variables.md

```diff
@@ -10,6 +10,13 @@ The env vars are only applicable to the Jackson service. If you are using the np
 
 ## General configuration
 
+### **HOSTNAME**
+
+The container network that the service starts listening on. 
```

Comment:
Could we simplify this to:

If you need to bind to a specific hostname other than localhost, you can define `HOSTNAME` environment variable. For example `HOSTNAME=0.0.0.0`.

Reply:
That's the problem, without `HOSTNAME=0.0.0.0` the service does not listen to localhost and it's not reachable on `http://localhost:5225/api/health` making the health-check fail.

You can reproduce with:
```
version: "3.3"
services:
  jackson:
    ports:
      - 5225:5225
    platform: linux/x86_64
    environment:
      # - HOSTNAME=0.0.0.0
      - DB_ENGINE=sql
      - DB_TYPE=postgres
      - DB_SSL=true
      - DB_SSL_REJECT_UNAUTHORIZED=false
      - DB_URL=postgres://postgres:asecretpassword@postgres:5432/sso_staging
      - DB_ENCRYPTION_KEY=notSoSecretDatabaseEncryptionKey
      - JACKSON_API_KEYS=secretApiKey594
      - NEXTAUTH_URL=https://localhost:5225
      - EXTERNAL_URL=https://localhost:5225
      - NEXTAUTH_SECRET=NotSecretAtAllNextAuthSecret
      - NEXTAUTH_ADMIN_CREDENTIALS=email@example.com:averysecretpassword
      - SAML_AUDIENCE=https://olivahealth.app
      - WEBHOOK_URL=http://localhost:3000/api/hooks/sso
      - WEBHOOK_SECRET=averysecretpassword
      - IDP_ENABLED=true
    image: boxyhq/jackson:1.14.2
    healthcheck:
      test: curl -f http://localhost:5225/api/health || exit 1
      interval: 15s
      retries: 3
      start_period: 10s
      timeout: 10s
  postgres:
    image: postgres:16
    ports:
      - 5432:5432
    environment:
      - POSTGRES_PASSWORD=asecretpassword
      - POSTGRES_DB=sso_staging
```

Without `HOSTNAME`:
![image](https://github.com/boxyhq/website/assets/695180/7c5f6695-e97b-4d69-90b1-f3a52865bcb3)

With `HOSTNAME=0.0.0.0`
![image](https://github.com/boxyhq/website/assets/695180/155a15f8-7cad-42f3-b8ee-3f3b66d4672f)

I don't know why the Docker image `boxyhq/jackson:1.14.2` by default provides the `HOSTNAME` like this:
![image](https://github.com/boxyhq/website/assets/695180/6827f769-79b4-4437-9d71-4f6852876e5f)

This is the first service that I use that does not listen to localhost by default 😅

Reply:
@sergio-toro I agree with the issue, I just meant the tweak to the text in the docs.